crates/ide-db/src/prime_caches.rs

```diff
@@ -68,6 +71,23 @@ pub fn parallel_prime_caches(
                     db.import_map(crate_id);
                 }
 
+                // Compute the symbol search index.
+                // This primes the cache for `ide_db::symbol_index::world_symbols()`.
+                //
+                // We do this for workspace crates only (members of local_roots), because doing it
+                // for all dependencies could be *very* unnecessarily slow in a large project.
+                //
+                // FIXME: We should do it unconditionally if the configuration is set to default to
+                // searching dependencies (rust-analyzer.workspace.symbol.search.scope), but we
+                // would need to pipe that configuration information down here.
+                //
+                // FIXME: This would complete faster if it were done in parallel as a separate
+                // work item. That will make the progress calculation more complex since there
+                // will be more than one work item per crate.
```

Comment:
it wouldn't, this depends on the definition maps which is what we are already computing here, so if we were to run this in parallel it would just block on that computation.

Reply:
OK, I deleted this comment.